libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.ts

```diff
@@ -44,6 +45,13 @@ export class SkyCheckboxGroupComponent {
   @Input({ transform: booleanAttribute })
   public labelHidden = false;
 
+  /**
+   * Indicates whether the checkbox group is required.
```

Comment:
Probably should start with "Whether" like (most) of our other boolean inputs.
"Whether the checkbox group is required."

Reply:
Updated